code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1261,5 +1261,19 @@
 	if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
 		air.temperature = max(air.temperature * heat_capacity / new_heat_capacity, TCMB)
 
+	// Rusts its surroundings.
+	if(SPT_PROB(1, antinoblium_moles / 100) && isturf(holder)) // 1% chance per tick at 100 moles.
+		var/turf/location = holder
+		var/list/turfs = list(location)
+		for(var/direction in GLOB.cardinals)
+			var/turf/turf = get_step(location, direction)
+			if(isclosedturf(turf))
+				turfs += turf
+		for(var/turf/turf in turfs)
+			if(!HAS_TRAIT(turf, TRAIT_RUSTY)) // prevents destroying walls
+				turf.rust_turf()
+			for(var/obj/obj in turf)
+				obj.rust_heretic_act()
```

Comment:
This will instakill most machinery and structures, as they just straight up take 500 damage from it. Did you even test this?

Reply:
I decided to just not rust objects as I realised it will make fixing it almost impossible even if you contained it.